org.mpo.Dayon.appdata.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+    <id>ch.galante.Dayon</id>
```

Comment:
this domain redirect to some Google in chinese...

Given that the website is https://retgal.github.io/Dayon/ , `io.github.retgal.Dayon` seems to be the appropriate one

Reply:
galante.ch *is* my domain! (it's in fact my familyname..)

Reply:
*upstream project* is part of the requirements.

Reply:
But I really don't care, `io.github.retgal` is also fine..